### PR TITLE
Register new package Microeconometrics.jl v0.0.1

### PR DESCRIPTION
**Pull request by lbittarello**
Repository: https://github.com/lbittarello/Microeconometrics.jl

### CODE REVIEW

**Comment by lbittarello** [comment]
I'd forgot to end the name of the repository with ".jl"... Fixed!

File: Microeconometrics/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,6 @@
+julia 0.6-
```
Comment (tkelman) [inline Microeconometrics/versions/0.0.1/requires]: this includes the very first dev version after 0.5 branched, so `mutable struct` syntax wouldn't work there - better to use `0.6.0-pre` as the minimum

Comment (lbittarello) [inline Microeconometrics/versions/0.0.1/requires]: Fixed!

Comment (tkelman) [inline Microeconometrics/versions/0.0.1/requires]: you'll need to update this pr. consider using https://github.com/attobot/attobot

**Pull request closed without merging**